src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(new Lottery().getRandomBall());
+        }
     }
 }
+
```

Comment:
redundant empty line, should be only 1 at the end of the file
Hint: check in the IDE because github shows it a bit different 😉